yb-voyager/src/query/queryissue/issues_ddl.go

```diff
@@ -540,6 +555,10 @@ var deterministicOptionCollationIssue = issue.Issue{
 	Suggestion:  "This feature is not supported in YugabyteDB yet",
 	GH:          "https://github.com/yugabyte/yugabyte-db/issues/25575",
 	DocsLink:    "https://docs.yugabyte.com/preview/yugabyte-voyager/known-issues/postgresql/#postgresql-12-and-later-features",
+	MinimumVersionsFixedIn: map[string]*ybversion.YBVersion{
+		ybversion.SERIES_2_25: ybversion.V2_25_0_0, // TODO: understand why docs says coming soon non-deterministic collation but the CREATE COLLAITON works on 2.25
```

Comment:
If we don't have clarity, i would err on the side of reporting the issue, and then later fixing it, rather than marking it as fixed.

Reply:
As discussed in standup, anything is fine, since you have already marked them as fixed, it should be okay 👍